<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reseau de communication</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="sidebarWidget"><layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="placeholderText">
           <string>Search contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Research</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
        <widget class="QPushButton" name="addButton">
          <property name="text">
            <string>Add contact</string>
          </property>
        </widget>
      </item>
      <item>
       <widget class="QListWidget" name="contactList">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
     </layout></widget>
    </item>
    <item>
     <widget class="QWidget" name="chatWidget"><layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="contactNameLabel">
        <property name="text">
         <string>Choisir un contact</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="chatDisplay"/>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="messageLineEdit">
          <property name="placeholderText">
           <string>Type a message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout></widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
